package.json

```diff
@@ -51,7 +54,8 @@
   "prettier": "@spotify/prettier-config",
   "lint-staged": {
     "*": "yarn run prettier:fix",
-    "*.{js,jsx,ts,tsx,mjs,cjs}": "yarn run lint -- -- --fix"
+    "*.{jsx,ts,tsx,mjs,cjs}": "yarn run lint -- -- --fix",
+    "!(.eslintrc).js": "yarn run lint -- -- --fix"
```

Comment:
this syntax for "all `.js` except `.eslintrc.js` looks weird to me, but if you've tested and it works, then +1.

Reply:
Why does `.eslintrc.js` need to be excluded? I'm fine excluding it, I'm just a little curious to what the error is.

Reply:
Because without it, the `yarn lint-staged` command fails (in a number of case, according to what you stage for commit) with many errors like the following:

```
app:lint: 
app:lint: 
app:lint:   ⚠  http://eslint.org/docs/rules/
app:lint: 
app:lint:      File ignored by default.  Use a negated ignore pattern (like "--ignore-pattern '!<relative/path/to/filename>'") to override
app:lint: 
app:lint: 
app:lint:      dynamic-plugins/backstage-plugin-aap-backend-wrapped/.eslintrc.js:0:0
app:lint: 
app:lint: ✘ 1 problem (0 errors, 1 warning)
app:lint: 
app:lint: 
app:lint: Warnings:
app:lint:   1  http://eslint.org/docs/rules/null
app:lint: 
```